src/jsp/com/cubrid/plcsql/compiler/SymbolStack.java

```diff
@@ -290,6 +290,10 @@ DeclProc getDeclProc(String name) {
         return getDecl(DeclProc.class, name);
     }
 
+    DeclVar getDeclVar(String name) {
```

Comment:
This implementation of getDeclVar() cannot be used because an Id expression can also be a function call expression without any arguments (Oracle and CUBRID syntax allow it) For an Id, we have to look-up symbol table both for Ids and for functions. Exisiting getDeclForIdExpr() is for this purpose.
For example, we have to raise an semantic error in the following case:

`...
    i int;
begin
    declare
        function i ...
    begin
        select ... where col = i;    -- i cannot be a function call in Static SQL in Oracle and CUBRID
    end;
    ...
end;
`